TrailBlaze/View Controllers/HomeViewController.m

```diff
@@ -202,6 +229,7 @@ - (MKOverlayRenderer *)mapView:(MKMapView *)mapView rendererForOverlay:(id<MKOve
 
 - (void)locationManager:(CLLocationManager *)manager didUpdateLocations:(NSArray *)locations {
     currentLocation = [locations lastObject];
+    [pacer paceTracker:pacer.polylinePoints userLocation:self->currentLocation bestPaces:pacer.bestPacesDictionary];
```

Comment:
we're passing properties of the pacer back into itself in this call. is it ever necessary to pass external values? if not this API can probably be modified. also another thing which supports making the currently public properties into private.

Reply:
Have edited the methods to use their instances properties instead of taking them in again as parameters, that was very awful OOP😅